geomesa-utils/src/main/scala/org/locationtech/geomesa/utils/geotools/Conversions.scala

```diff
@@ -191,6 +193,9 @@ object RichSimpleFeatureType {
   val DEFAULT_DATE_KEY    = "geomesa.index.dtg"
   val ST_INDEX_SCHEMA_KEY = "geomesa.index.st.schema"
   val USER_DATA_PREFIX    = "geomesa.user-data.prefix"
+  val KEYWORDS_KEY        = "geomesa.keywords"
+
+  val KEYWORDS_DELIMITTER = """\|"""
```

Comment:
Not sure - is there a specific use case in mind for this? Presumably these are going to be added to e.g. an SFT conf file? Seems like for e.g. stealth, there aren't any commas